### DISCUSSION TITLE
skywalking-php agent mysqli extension doesn't work

### DISCUSSION THREAD

**Discussion by thecalmund** [apache/skywalking#10041] [Q&A]
Hello,

i currently have a Skywalking setup running on the latest version with the latest release (0.2.0) of the agent and an application that can use PDO or mysqli as mysql driver. With PDO I get traces in Skywalking and when I change the mysql driver to mysqli I don't get traces anymore. Does anyone here have the same problem?

**Comment by thecalmund** [comment]
Does anyone have an idea?

**Comment by jmjoy** [comment]
Please show your logs, configure as `skywalking_agent.log_file` in your `php.ini`.

**Comment by thecalmund** [comment]
[skywalking_Log.log](https://github.com/apache/skywalking/files/10133221/skywalking_Log.log)

**Reply by jmjoy** [reply]
I don't found any `mysqli` call in the log, can you show your codes?
